doc/source/_templates/layout.html

```diff
@@ -1,35 +1,22 @@
 {% extends "!layout.html" %}
 
 {%- block extrahead %}
-<style>
-  .main {
-    -moz-box-shadow: none;
-    -webkit-box-shadow: none;
-    box-shadow: none;
-  }
-  div.top-scipy-org-logo-header {
-    background-color: #fafafa;
-    border-bottom: 2px solid #013243; /* Warm Black */
-    margin-top: 0;
-    box-shadow: none;
-  }
-  div.top-scipy-org-logo-header img {
-    height: 100px;
-    padding-left: 27px;
-  }
-  div.spc-navbar .nav-pills > li > a {
-    background-color: #4d77cf; /* Han Blue */
-  }
-</style>
```

Comment:
Can you move this code around in a standalone PR?

Reply:
This code is already in #16988. I've moved it out of the header. This is not two PRs in one.

Reply:
I'm suggesting making a PR that does nothing but move this code. It makes your other PRs conflict less, and makes all the PRs easier to review.

Reply:
This makes sense and I'll bear it in mind for future PRs.